### PR TITLE
added in copy and paste for obligated sent on

### PR DESCRIPTION
**Pull request by willdot**
PBI: 69684

Added the copy paste functionality to the obligated sent on for AATFs

### CODE REVIEW

**Comment by JohnPadgettCivica** [comment]
is it possible to re-use the ObligatedValuesCopyPasteController and add a redirect so we know where to redirect back to? Will make sure that if we need to make changes to this copy and paste we only need to make it in the one place...

**Comment by willdot** [comment]
> is it possible to re-use the ObligatedValuesCopyPasteController and add a redirect so we know where to redirect back to? Will make sure that if we need to make changes to this copy and paste we only need to make it in the one place...

Attempted that yesterday, but got messy because the different controllers accept different parameters to the action methods.

File: src/EA.Weee.Web/Areas/AatfReturn/Controllers/ObligatedSentOnController.cs [hunk 1]
```diff
@@ -50,7 +50,8 @@ public virtual async Task<ActionResult> Index(Guid returnId, Guid organisationId
                     ReturnData = @return,
                     AatfId = aatfId,
                     OperatorName = operatorName,
-                    WeeeSentOnId = weeeSentOnId
+                    WeeeSentOnId = weeeSentOnId,
+                    PastedData = TempData["pastedValues"] as ObligatedCategoryValue
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/AatfReturn/Controllers/ObligatedSentOnController.cs]: Can we add a unit test to make sure that if the pasted values are provided they are set on the mapper object? ta...

Comment (willdot) [reply src/EA.Weee.Web/Areas/AatfReturn/Controllers/ObligatedSentOnController.cs]: Just added a test.

**Pull request merged**